shard-operator/pom.xml

```diff
@@ -59,6 +59,12 @@
       <groupId>io.fabric8</groupId>
       <artifactId>openshift-model-monitoring</artifactId>
     </dependency>
+    <!-- needed for https://issues.redhat.com/browse/MGDOBR-503 https://github.com/quarkusio/quarkus/issues/24644 -->
+    <dependency>
+      <groupId>io.quarkus</groupId>
+      <artifactId>quarkus-minikube</artifactId>
```

Comment:
I don't think we want to include this when we build the artifacts for OCP. This is why it was added only with the `minikube` profile